packages/plugins/rubygem-smart_proxy_salt_core/rubygem-smart_proxy_salt_core.spec

```diff
@@ -80,6 +84,9 @@ cp -pa .%{gem_dir}/* \
 
 
 %changelog
+* Tue Jan 07 2020 Eric D. Helms <ericdhelms@gmail.com> - 0.0.2-2
+- Add obsoletes on non-SCL version of the package.
```

Comment:
I've added this so that Jenkins is happy. Technically we don't have to rebuild the *_core packages, as they are in the SCL already.